recipes/h3/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -ex
+
+cmake \
+  -DENABLE_FORMAT=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_SKIP_INSTALL_ALL_DEPENDENCY=ON \
+  .
+
+make h3 geoToH3 h3ToComponents h3ToGeo h3ToGeoBoundary hexRange kRing h3ToGeoBoundaryHier h3ToGeoHier h3ToHier -j${CPU_COUNT} VERBOSE=1
```

Comment:
Can you change this `make -k -j${CPU_COUNT} | true` and then run the testsuite? (benchmarks will fail, but other targets will be built)

Reply:
@isuruf I might have misunderstood you, but the last change fails in CI with:
```
+ make -k -j2
make[2]: *** [CMakeFiles/h3.dir/build.make:311: CMakeFiles/h3.dir/depend] Error 141
make[1]: *** [CMakeFiles/Makefile2:5122: CMakeFiles/h3.dir/all] Error 2
make[1]: Target 'all' not remade because of errors.
make: *** [Makefile:141: all] Error 2
make: Target 'default_target' not remade because of errors.
+ make install
CMake Error at cmake_install.cmake:47 (file):
  file INSTALL cannot find
  "/home/conda/feedstock_root/build_artifacts/h3_1560169178512/work/lib/libh3.so.1".
```